WELD.md

```diff
@@ -139,3 +104,44 @@ python weld.py
 ### Goal:
 
 1.  Track the submodule (recipe) version with the current data repository.
+
+## Step 5: Commit most weld results to Github
+
+### Procedure:
+
+Depending on the size of your dataset, it may not be practical to commit all weld results to Github.
+Our team has decided to commit all summary figures and results but not the per-site paint or spots data to Github.
+You can tailor the following commit code to suit your data storage preferences.
+
+```bash
+# EDIT {BATCH} IN FOLLOWING CODE TO MATCH YOUR DATASET
+# Add, commit, and push the weld results
+git add data/0.site-qc/{BATCH}/data
+git add data/0.site-qc/{BATCH}/figures
+git add data/0.site-qc/{BATCH}/results
+git add data/1.profiles
```

Comment:
does this step add single cell profiles?

I think we are only adding bulk profiles here - still tbd what to do about single cell (likely just s3 for now)